nc/include/component/AutoComponentGroup.h

```
@@ -45,17 +46,15 @@ namespace nc
     };
 
     template<std::derived_from<AutoComponent> T, class ... Args>
-    T* AutoComponentGroup::Add(Args&& ... args) noexcept
+    T* AutoComponentGroup::Add(Args&& ... args)
```

Always nice to throw instead of return null IMO, nice